etc/release_manifests/EXAMPLE-ML-DGA-20201111-1.json

```diff
@@ -0,0 +1,58 @@
+{
+  "assets": {
+    "ML-DGA-20201111-1.zip": {
+      "created_at": "2020-11-11T22:32:18Z",
+      "description": "ML-DGA-20201111-1.zip",
+      "entries": {
+        "dga_875109_1.0_dga_ngrams_create.json": {
+          "compress_size": 538,
+          "created_at": "2020-11-11T13:32:14Z",
+          "name": "dga_875109_1.0_dga_ngrams_create.json",
+          "sha256": "4eb88a59b97b01459e7b239e31d75ec9cbf9fa0d68603e3cd624cd937c3a2a11",
+          "size": 1779
+        },
+        "dga_875109_1.0_dga_ngrams_transform_delete.json": {
+          "compress_size": 229,
+          "created_at": "2020-11-11T13:32:14Z",
+          "name": "dga_875109_1.0_dga_ngrams_transform_delete.json",
+          "sha256": "a3a2e3ab0874950e501e25229766b4898a264514e33977fd4ca58c135a5491a7",
+          "size": 605
+        },
+        "dga_875109_1.0_ingest_pipeline1.json": {
+          "compress_size": 146,
+          "created_at": "2020-11-11T13:32:14Z",
+          "name": "dga_875109_1.0_ingest_pipeline1.json",
+          "sha256": "670cf7573cbf0511a9af2165cf1f6cd9f955a4942b0cdef8b8edcf9737a0abc6",
+          "size": 204
+        },
+        "dga_875109_1.0_ingest_pipeline2.json": {
+          "compress_size": 234,
+          "created_at": "2020-11-11T13:32:14Z",
+          "name": "dga_875109_1.0_ingest_pipeline2.json",
+          "sha256": "32b7d4a6c6ac1688169ff7b18fe9102e703083de598bf226ff8a0792f4bb7b17",
+          "size": 435
+        },
+        "dga_875109_1.0_model.json": {
+          "compress_size": 18086240,
+          "created_at": "2020-11-11T13:32:14Z",
+          "name": "dga_875109_1.0_model.json",
+          "sha256": "897ff6135a593ab7bf8bfa29df8d9643372fd7eba835c54aec1c60b3aef4eb77",
+          "size": 23894228
+        }
+      },
+      "relative_url": "https://github.com/brokensound77/detection-rules/releases/download/ML-DGA-20201111-1/ML-DGA-20201111-1.zip",
+      "zipped_sha256": "4ce9439b21e9601e71e849bde842cb0c65760b40aff53ef6df0f04d2ae6fde84"
+    }
+  },
+  "assets_url": "https://api.github.com/repos/brokensound77/detection-rules/releases/33828196/assets",
+  "author": "brokensound77",
+  "created_at": "2020-10-30T17:19:50Z",
+  "description": "model_name: dga_875109_1.0\n\n----\n\n\ndate: 2020-11-11T13:32:15Z\nmodel_sha256: 897ff6135a593ab7bf8bfa29df8d9643372fd7eba835c54aec1c60b3aef4eb77\n\n----\n\nExtra description info",
+  "html_url": "https://github.com/brokensound77/detection-rules/releases/tag/ML-DGA-20201111-1",
+  "id": 33828196,
+  "name": "ML-DGA-20201111-1",
+  "published_at": "2020-11-11T22:32:18Z",
+  "tag_name": "ML-DGA-20201111-1",
+  "url": "https://api.github.com/repos/brokensound77/detection-rules/releases/33828196",
+  "zipball_url": "https://api.github.com/repos/brokensound77/detection-rules/zipball/ML-DGA-20201111-1"
```

Comment:
do you think the manifest should contain any info beyond what'll be in the zip file?

Reply:
what'd you have in mind?